xclim/testing/tests/test_calendar.py

```diff
@@ -377,3 +379,42 @@ def test_datetime_to_decimal_year(source_cal, exp180):
     )
     decy = datetime_to_decimal_year(times, calendar=source_cal)
     np.testing.assert_almost_equal(decy[180] - 2004, exp180)
+
+
+def test_doy_to_days_since():
+    # simple test
+    time = date_range("2020-07-01", "2022-07-01", freq="AS-JUL")
+    da = xr.DataArray([190, 360, 3], dims=("time",), coords={"time": time})
+
+    out = doy_to_days_since(da)
+    np.testing.assert_array_equal(out, [7, 178, 186])
+
+    da2 = days_since_to_doy(out)
```

Comment:
line 392 :  same calculation with the valid `start` via `doy_to_days_since(da,start='07-01')`   gives different results from original `da = [190, 360, 3]`  but `da2 = [190, 360, 368]`

Reply:
Woups. That's a ">" that should be a ">=" (calendar.py line 848)...

Reply:
Just figured this out too!